tests/integer_traits.rs

```diff
@@ -0,0 +1,210 @@
+use std::ffi::{c_long, c_ulong};
```

Comment:
These aliases were added in Rust 1.64, but this project supports back to 1.61. They are available in 1.61, just under a different module.

```suggestion
use std::os::raw::{c_long, c_ulong};
```